ui/src/components/layouts/SiderLayout.tsx

```diff
@@ -1,30 +1,73 @@
-import { Layout } from "antd";
+import { Grid, Layout, Row } from "antd";
+import { useState } from "react";
 import { Outlet } from "react-router-dom";
 
+import { ReactComponent as IconMenu } from "src/assets/icons/menu-01.svg";
 import { FeedbackLink } from "src/components/shared/FeedbackLink";
+import { LogoLink } from "src/components/shared/LogoLink";
 import { SiderMenu } from "src/components/shared/SiderMenu";
 import { SIDER_WIDTH } from "src/utils/constants";
 
 export function SiderLayout() {
+  const [collapsed, setCollapsed] = useState(true);
+
+  const { lg } = Grid.useBreakpoint();
+
   return (
     <Layout>
+      {!lg && (
+        <Layout.Header
+          style={{
+            borderBottom: "1px solid #EAECF0",
+            height: 64,
+            padding: "20px 20px",
+            position: "fixed",
+            top: 0,
+            width: "100%",
+            zIndex: 3,
+          }}
+        >
+          <Row justify="space-between">
+            <LogoLink />
+            <IconMenu onClick={() => setCollapsed(!collapsed)} />
```

Comment:
The menu icon is not properly centered. This is due to the padding set in the Header, which is 20px. The height of the header is 64px, and if you subtract 40px (padding top + bottom), the space left for the icon is 24px, but the real height of the icon is 28px, so it runs out of space and makes it not being centered properly:

<img width="551" alt="Captura de pantalla 2023-05-11 a las 19 55 41" src="https://github.com/0xPolygonID/issuer-node/assets/6737805/9d9335da-6544-4a7b-b10b-344cf3daf3f1">

Apart from that I'd wrap the svg into a `button` so it has `cursor: pointer` CSS prop to match the rest of the buttons in the app.

Reply:
I wrapped it into a button but it gives more padding and lose more control over the position. 
I'm going to try a pixel-perfect version

Reply:
Now it looks perfect!